src/main/java/domain/Money.java

```diff
@@ -0,0 +1,24 @@
+package domain;
+
+public class Money{
+    private long money;
```

Comment:
`final` 로 선언할 수 있을 것 같아요!

```suggestion
    private final long money;
```

Reply:
확실히 final로 재대입을 막는게 좋아 보입니다 👍